#
# Ubuntu: https://bugs.launchpad.net/ubuntu/+source/epiphany-browser/+bug/302859
# Upstream: none
# Description: Don't try to use a Gecko higher than 1.9.0.*
#
---
 embed/mozilla/mozilla-embed-single.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: epiphany-browser-2.24.1/embed/mozilla/mozilla-embed-single.cpp
===================================================================
--- epiphany-browser-2.24.1.orig/embed/mozilla/mozilla-embed-single.cpp
+++ epiphany-browser-2.24.1/embed/mozilla/mozilla-embed-single.cpp
@@ -585,17 +585,17 @@
 
 #ifdef HAVE_GECKO_1_9
 	NS_LogInit ();
 #endif
 
 #ifdef XPCOM_GLUE
 	static const GREVersionRange greVersion = {
 	  "1.9a", PR_TRUE,
-	  "2", PR_TRUE
+	  "1.9.0.*", PR_TRUE
 	};
 	char xpcomLocation[4096];
 	nsresult rv = GRE_GetGREPathWithProperties(&greVersion, 1, nsnull, 0, xpcomLocation, sizeof (xpcomLocation));
 	if (NS_FAILED (rv))
 	{
 	  g_warning ("Could not find a suitable GRE!\n");
 	  return FALSE;
 	}
